### PR TITLE
Configure docs to enable versioning at a future time

### PR DESCRIPTION
**Pull request by tpdorsey**
Adds configuration to enable version switching in mkdocs.yml.

Corrects image urls so images appear when versions are in use.

### Checklist

- [ ] This pull request references any related issue by including "closes `<link to issue>`"
	- If no issue exists and your change is not a small fix, please [create an issue](https://github.com/PrefectHQ/prefect/issues/new/choose) first.
- [x] This pull request includes tests or only affects documentation.
- [ ] This pull request includes a label categorizing the change e.g. `fix`, `feature`, `enhancement`

### CODE REVIEW

File: mkdocs.yml [hunk 1]
```diff
@@ -73,6 +73,7 @@ theme:
             icon: material/weather-night
             name: Switch to light mode
 plugins:
+    mike:
```
[the remark this answers is not in the document]
Comment (tpdorsey) [reply mkdocs.yml:76]: no it works fine as is. mkdocs just needs to know it's enabled

File: docs/tutorials/storage.md [hunk 2]
```diff
@@ -69,17 +69,17 @@ Most users will find it easiest to configure new storage blocks through the Pref
...
 
-![Viewing a list of block types in the Prefect UI](/img/tutorials/choose-blocks.png)
+![Viewing a list of block types in the Prefect UI](../img/tutorials/choose-blocks.png)
 
 For this tutorial example, we'll use the AWS S3 block as an example. If you use a different cloud storage service or solution, feel free to use the appropriate block type. The process is similar for all blocks, though the configuration options are slightly different, reflecting the authentication requirements of different cloud services.
 
 Scroll down the list of blocks and find the **S3** block, then select **Add +** to configure a new storage block based on this block type. Prefect displays a **Create** page that enables specifying storage settings.
 
-![Configurating an S3 storage block in the Prefect UI](/img/tutorials/s3-block-configuration.png)
+![Configurating an S3 storage block in the Prefect UI](../img/tutorials/s3-block-configuration.png)
```
Comment (serinamarie) [inline docs/tutorials/storage.md:82]:
```suggestion
![Configuring an S3 storage block in the Prefect UI](../img/tutorials/s3-block-configuration.png)
```

File: docs/concepts/storage.md [hunk 3]
```diff
@@ -68,11 +68,11 @@ You can [create, edit, and manage storage blocks](/ui/blocks/) in the Prefect UI
 
 To create a new block, select the **+** button. Prefect displays a library of block types you can configure to create blocks to be used by your flows.
 
-![Viewing the new block library in the Prefect UI](/img/ui/orion-block-library.png)
+![Viewing the new block library in the Prefect UI](../img/ui/orion-block-library.png)
 
 Select **Add +** to configure a new storage block based on a specific block type. Prefect displays a **Create** page that enables specifying storage settings.
 
-![Configurating an S3 storage block in the Prefect UI](/img/tutorials/s3-block-configuration.png)
+![Configurating an S3 storage block in the Prefect UI](../img/tutorials/s3-block-configuration.png)
```
Comment (serinamarie) [inline docs/concepts/storage.md:75]:
```suggestion
![Configuring an S3 storage block in the Prefect UI](../img/tutorials/s3-block-configuration.png)
```

File: docs/ui/organizations.md [hunk 4]
```diff
@@ -86,7 +86,7 @@ The maximum number of organization members varies. See the [Prefect Cloud plans]
 
 Select **Service Accounts** to view, create, or edit service accounts for your organization.
 
-![Viewing service accoutns for an organization in Prefect Cloud.](/img/ui/service-accounts.png)
+![Viewing service accoutns for an organization in Prefect Cloud.](../img/ui/service-accounts.png)
```
Comment (serinamarie) [inline docs/ui/organizations.md:89]:
```suggestion
![Viewing service accounts for an organization in Prefect Cloud.](../img/ui/service-accounts.png)
```

File: docs/ui/notifications.md [hunk 5]
```diff
@@ -54,4 +54,4 @@ When the conditions of the notification are triggered, you’ll receive a simple
 
 On the **Notifications** page you can pause, edit, or delete any configured notification.
 
-![Viewing all configured notification in the Prefect UI](/img/ui/orion-notifications.png)
+![Viewing all configured notification in the Prefect UI](../img/ui/orion-notifications.png)
```
Comment (serinamarie) [inline docs/ui/notifications.md:57]:
```suggestion
![Viewing all configured notifications in the Prefect UI](../img/ui/orion-notifications.png)
```

File: docs/ui/flow-runs.md [hunk 6]
```diff
@@ -31,27 +31,27 @@ The following sections discuss each section of the dashboard view.
...
 
-![Highlighting the details section of the flow runs page.](/img/ui/orion-dash-details.png)
+![Highlighting the details section of the flow runs page.](../img/ui/orion-dash-details.png)
 
 The default view shows all flow runs observed by the Prefect API and scheduled flow runs for deployments. You can use filters to display only the flow runs that meet your filter criteria, such as status or tags.
 
 Each row in the flow run details shows information about a specific completed, in progress, or scheduled flow run.
 
-![Information displayed for a flow run in the UI](/img/ui/orion-flow-run-examples.png)
+![Information displayed for a flow run in the UI](../img/ui/orion-flow-run-examples.png)
```
Comment (serinamarie) [inline docs/ui/flow-runs.md:54]:
```suggestion
![Information displayed for a flow run in the Prefect UI](../img/ui/orion-flow-run-examples.png)
```

**Comment by madkinsz** [comment]
Should I see any changes in the deploy preview?

File: docs/concepts/storage.md (hunk 3 above)
Comment (tpdorsey) [reply docs/concepts/storage.md:75]: lol good catch.

**Comment by tpdorsey** [comment]
@madkinsz you should not see any difference at this point

**Pull request merged**